### ISSUE TITLE
[Issue]: Add accessible names (button labels) for official iOS client

### ISSUE THREAD

**Issue by charmarkk** [jellyfin/jellyfin-web#3215]
### Please describe your bug

I have a mobility impairment and use Voice Control for iOS to interact with my iPad most often. There are two ways to access on-screen elements with Voice Control, numbers (which iOS assigns to visual elements) and names (which are taken from the labels of the elements in an app, webpage, etc. as assigned by developers).

See [Deque's](https://www.deque.com/blog/new-in-ios-13-accessibility-voice-control-and-more/) and [Apple's](https://support.apple.com/en-us/HT210417) articles on the topic for more about this really cool accessibility feature!

I usually use the numbers option, as it's usually a bit easier and more reliable than guessing what the accessible name for an object is, or switching voice control to display those, but with media server clients this is tricky since the numbers always change. It can be hard to get them to show up as well, since you have to "tap" the screen to get the player controls to show up, then Voice Control has to scan the objects on screen and assign them numbers. Once it does this, the player control hide delay expires and the controls disappear, making it hard to "tap" an item by number. It's much easier to say "tap pause" once the controls come up.

I've used Plex for ages, but wanted to switch to something more feature rich and open source, but Jellyfin (for iOS at least) appears to be lacking several crucial accessibility features. I could use "tap play" (or whatever) with Plex for the player controls, but it did not work with Jellyfin. When I invoked the "Show names" Voice Control command, it only showed a unicode "?" ( ⍰ ) for most interactive elements on the app, leading me to believe the player control, menu, and all other icons are made with icon fonts or emojis, and their labels are that icon itself (or non-standard emoji) rather than a human readable string.

What should happen is each button should have an [accessible name](https://www.w3.org/TR/wai-aria-1.2/#namecalculation) - a human readable and understandable label (like "Play", "Skip Forward", "Menu", etc.) - which is what would then be made to be interactive when I say "Tap Play", "Tap Skip Forward", etc. and what a blind/low-vision user using VoiceOver would hear when navigating the app via touch. As is, they would not hear anything. 

I understand this may not be the particularly correct repo to log this issue, but I saw no other one for iOS except Swiftfin, which seemed different than what is [currently on Apple's app store](https://apps.apple.com/us/app/jellyfin-mobile/id1480192618), especially since it is only in TestFlight. I'd love to help with this any way I can, but I am not an iOS or Swift dev - I can do HTML and rudimentary JS (both of which are where most web accessibility lives). Please direct me to a better repo to file an issue, if this isn't a good place. However, this is (in principle) a simple fix. 

[This is also a requirement set forth by the W3C's Web Content Accessibility Guidelines](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_overview#name-role-value) and [Apple's App Store requirements](https://developer.apple.com/app-store/review/guidelines/#before-you-submit) (Before You Submit > Dev Guidelines > UIKit > [Accessibility for UIKit](https://developer.apple.com/documentation/uikit/accessibility_for_uikit)). FWIW, Apple fully documents how iOS apps, even with SwiftUI, should use iOS' accessibility API.

I will try to get some screenshots of the issue I'm experiencing to add so it is better illustrated.

Please let me know how I may help! :) I'm happy to learn.

Edit to add: This is likely an issue throughout the app, Android app (Android added Voice Control in Android 11), and other clients - I just happened to notice it with iOS.

12/09 Edit to add: I included screenshots for the accessible names from both the Jellyfin app and Plex (for comparison). If you look close, the Jellyfin shuffle button has an accessible name - again, what I would have to say "Tap XXX" to activate - of a silverware emoji. I can't even begin to imagine how to say that in English. 😄 That's probably due to the way iOS tries to translate icon fonts and similar things, but it's a great example of the problem!

### Jellyfin Version

10.7.7

### Environment

```markdown
- OS: Ubuntu Server
- Virtualization:
- Clients: iOS Client
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Please attach any screenshots here

![photo_2021-12-09_11-09-31](https://user-images.githubusercontent.com/22264269/145443208-41beb0ec-a91a-43f3-a2d3-6bec11512575.jpg)
![photo_2021-12-09_11-09-24](https://user-images.githubusercontent.com/22264269/145443239-a1bb4386-e67a-4784-aa3c-76a655c2dd55.jpg)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by thornbill** [comment]
We made an effort to substantially improve accessibility in the 10.8 server release. Please let us know how it is working and any areas where it is still lacking!

**Comment by charmarkk** [comment]
This is great news, I'll have to try it again. Thank you for the update and
taking accessibility seriously.

**Comment by charmarkk** [comment]
Hi @thornbill! It has been a bazillion years since I looked at this, and I got stuck in my ways with Plex. However, Plex has been frustrating me again lately, prompting me to check this out!

I'm happy to say that it looks like, at least everything I checked, has appropriate names and is working as expected with Voice Access on iOS now. Thank you so much!
